x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -162,24 +108,17 @@ export function OverviewPage({ routeParams }: Props) {
         </EuiFlexItem>
 
         {/* Alert section */}
-        {!!alerts.length && (
-          <EuiFlexItem grow={3}>
-            <AlertsSection alerts={alerts} />
-          </EuiFlexItem>
-        )}
+        <AlertsSection setAlertEmptySection={setAlertEmptySection} />
```

Comment:
IMO this callback is awkward. If this component needs to know about whether it has any data, I think this component should pass `AlertsSection` its data.